dev/ProgressBar/ProgressBar.cpp

```diff
@@ -121,7 +123,16 @@ void ProgressBar::SetProgressBarIndicatorWidth()
 
             if (IsIndeterminate())
             {
-                progressBarIndicator.Width(progressBarWidth * 0.4);
+                determinateProgressBarIndicator.Width(0);
+
+                if (auto&& indeterminateProgressBarIndicator = m_indeterminateProgressBarIndicator.get())
+                {
+                    if (auto&& indeterminateProgressBarIndicator2 = m_indeterminateProgressBarIndicator2.get())
```

Comment:
Setting indeterminateProgressBarIndicator's width shouldn't be dependent on the presense of an indeterminateProgressBarIndicator2

Reply:
@StephenLPeters There are two indicators, `indeterminateProgressBarIndicator` and `indeterminateProgressBarIndicator2`, running simultaneously for indeterminate mode as per design specifications